src/quo2/components/list_items/community_list/component_spec.cljs

```diff
@@ -0,0 +1,79 @@
+(ns quo2.components.list-items.community-list.component-spec
+  (:require [quo2.components.list-items.community-list.view :as component]
+            [test-helpers.component :as h]))
+
+(h/describe "Community list"
+  (h/test "default render"
```

Comment:
I would say this test does more than the default render, i.e it tests both press and on-long press events are triggered as well.